_locales/cs/maqueenPlusV2-jsdoc-strings.json

```diff
@@ -1,23 +1,52 @@
 {
     "maqueenPlusV2.controlMotor": "Blok pro ovládání motorů. Umožňuje měnit směr a rychlost otáček.",
-    "maqueenPlusV2.controlLED": "Blok pro ovládání předních LED světel.",
+    "maqueenPlusV2.controlMotor|param|emotor": "Identifikátor motoru z MyEnumMotor, např.: MyEnumMotor.LeftMotor",
```

Comment:
Not sure if I like the word `Identifikátor`. How about calling it `Volba motoru, např.: MyEnumMotor.LeftMotor`?

The same applies to `maqueenPlusV2.controlLED|param|eled` and others.

Reply:
Well, 'cause it's actually and ID of a motor... If I were to rephrase this one, I'd probably go with "Hodnota určující motor, pro který se mají nastavit směr a rychlost, např.: MyEnumMotor.LeftMotor" 

It's more descriptive, though much longer...